css/main.css

```diff
@@ -436,36 +307,40 @@ article h2 {
     letter-spacing: 1px;
 }
 
+.clearfix:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
 .print-icon {
-    float:right;
-    display:block;
-    width:55px;
-    height:10px;
-    margin:-23px -11px 13px 0;
+    width:100%;
 }
 
+
 .print-icon img{
+    display:inline;
+    float:right;
+    margin:74px -15px 0 0;
```

Comment:
Be careful of this high top margin-- You'll want to reduce it with a media query if you want to avoid interrupting the flow of the document at smaller widths. This is where flex's align-self: baseline; is a real boon.